src/cocotb/__init__.py

```diff
@@ -100,6 +101,17 @@ def _setup_logging() -> None:
 This is guaranteed to hold a value at test time.
 """
 
+packages: Optional[SimpleNamespace] = None
+"""A SimpleNamespace of package handles.
+
+This will be populated with handles at test time if packages can be discovered
+via the GPI.  Depending on the simulator, packages may need to be imported in
+the compliation unit scope or inside a module in order to be discoverable.
+Also note, the $unit pseudo-package is implemented differently between
+simulators.  It may appear as one or more attributes here depending on the
+number of compilation units.
```

Comment:
```suggestion
This will be populated with handles at test time if packages can be discovered
via the GPI.  Depending on the simulator, packages may need to be imported in
the compilation unit scope or inside a module in order to be discoverable.
Also note, the ``$unit`` pseudo-package is implemented differently between
simulators.  It may appear as one or more attributes here depending on the
number of compilation units.
```